source/reference/minio-mc/mc-replicate-diff.rst

```diff
@@ -0,0 +1,113 @@
+.. _minio-mc-replicate-diff:
+
+=====================
+``mc replicate diff``
+=====================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc replicate diff
+
+Description
+------
+
+.. start-mc-replicate-diff-desc
+
+The :mc:`mc replicate diff` shows object versions that are not replicated.
```

Comment:
```suggestion
The :mc:`mc replicate diff` shows object versions that are not replicated to the configured remote target.
```

We can configure multiple replication rules on a bucket right? If we have multiple remote arns, how does this work? Which remote do we check against, since prior to replication we have not yet determined which rule would hit (based on priority) 

Or does this only really work with site replication?

Reply:
`mc replicate` is bucket replication only, not site replication.
When the ARN is specified, the return is the replication status of the target ARN.
When not specified, the return is a general replication status of the bucket.

Added to the `arn` flag.